### PR TITLE
Feat/improve canvas samples

### PR DESCRIPTION
**Pull request by ccamel**
This PR brings some visual improvements to the canvas component storybook (for fun, more in the spirit of what we do ☺️).
I also took the opportunity to put the code of the demos in separate files.

image

### CODE REVIEW

**Review by JoeyDjr — APPROVED** [review]

**Comment by ccamel** [comment]
@fredericvilcot :pray:

**Review by fredericvilcot — COMMENTED** [review]
That's a really great work 😍
Just a little syntax remark, it's up to you ;)

**Review by lolottetheclash — DISMISSED** [review]
Such a nice work !
Just a typo remark on the `onCanvasCreated` method => `aftewards` which was here before this PR.

File: stories/atoms/components/canvas/bouncingText.ts [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+}
+
+export const drawBouncingText = (canvas: HTMLCanvasElement, deltaCount: number): void => {
+  const ctx = canvas.getContext('2d')
+
+  if (ctx) {
+    const text = `Size: ${canvas.width} / ${canvas.height}`
+    const metrics = ctx.measureText(text)
+
+    if (S.x == -1) {
```
[the remark this answers is not in the document]
Comment (ccamel) [reply stories/atoms/components/canvas/bouncingText.ts:18]: Sure! You are absolutely right. I will fix it that way. 👍

[the remark this answers is not in the document]
Comment (ccamel) [reply stories/atoms/components/canvas/bouncingText.ts]: Fixed with https://github.com/okp4/ui/pull/360/commits/d318a8b8d2013f146e1262e8f08cb555fbd89f0d.

**Review by ErikssonJoakim — APPROVED** [review]
Great initiative!

**Review by fredericvilcot — APPROVED** [review]

**Review by JoeyDjr — APPROVED** [review]

**Pull request merged**